### PR TITLE
Adding LICENSE file as a data file

### PR DESCRIPTION
**Pull request by davidcassany**
This commit includes the LICENSE file as a data_file in setup.py,
this way the source distribution tarball will also include the file.

### CODE REVIEW

**Comment by davidcassany** [comment]
@djungelorm gentle ping. Do you think the change is acceptable?

**Comment by djungelorm** [comment]
Yes this looks fine! Thanks. Sorry I've been a bit busy recently.

File: setup.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/python3
```
Comment (djungelorm) [inline setup.py]: Can we remove this hash bang? Would affect installing using python2

Comment (davidcassany) [reply setup.py]: Sure, gonna change it.

**Comment by davidcassany** [comment]
> Sorry I've been a bit busy recently.

No problem, thanks for looking at it.

**Pull request merged**